code-challenges.test.js

```diff
@@ -10,58 +10,210 @@
+
+// // b) Create the function that makes the test pass.
+
+// const sortObject = (obj) => {
+//   return Object.values(obj).sort((a, b) => a - b);
+//}
+
+//  [
+//   15, 15, 20, 30,
+//   30, 60, 90
+// ]
+
+//   at Object.log (code-challenges.test.js:101:11)
+
+// console.log
+// [
+//   10, 15,  20, 45,
+//   60, 65, 100
+// ]
+
+//   at Object.log (code-challenges.test.js:102:9)
+
+// PASS  ./code-challenges.test.js
+// sortObject
+// ✓ sorts the array of the values from least to greatest (19 ms)
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+
+
 
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
-// a) Create a test with expect statements for each of the variables provided.
+// Pseudocode:
+// input - create a test using a desribe function. Once the test works create a function
+// output - create a function that takes in an array and returns array of the accumulating sum. An empty array should return an empty array.
 
-const accountTransactions1 = [100, -17, -23, -9]
-// Expected output: [100, 83, 60, 51]
+// Create a test with the variable "accumlatingSum" and use the provided variables as the expect arguments
+// create a function using my variable accumalatingSum
+// Set a parameter in the function called arr
+// declare variable called result using let. result will return the results of the function into an array.
+// declare sum variable using let = 0
+// create a for loop that iterates thhrough  the array starting at index 0
+// create addition assignment for sum and array index
+// result.push(sum) 
+// return result which will return the accumalated sums in an array
+// console.log to see results in terminal
 
-const accountTransactions2 = [250, -89, 100, -96]
-// Expected output: [250, 161, 261, 165]
 
-const accountTransactions3 = []
-// Expected output: []
+// a) Create a test with expect statements for each of the variables provided.
 
-// b) Create the function that makes the test pass.
+// const accountTransactions1 = [100, -17, -23, -9]
+// // Expected output: [100, 83, 60, 51]
+
+// const accountTransactions2 = [250, -89, 100, -96]
+// // Expected output: [250, 161, 261, 165]
+
+// const accountTransactions3 = []
+// // Expected output: []
+// describe("accumulatingSum", () => {
+//   it("returns an array of the accumulating sum. an empty array should return an empty array", () => {
+//     expect(accumulatingSum(accountTransactions1)).toEqual([100, 83, 60, 51]),
+//     expect(accumulatingSum(accountTransactions2)).toEqual([250, 161, 261, 165]),
+//     expect(accumulatingSum(accountTransactions3)).toEqual([])
+//   })
+// })
+// // b) Create the function that makes the test pass.
+// const accumulatingSum = (arr) => {
+//   let result = [];
+//   let sum = 0;
+//   for (let i = 0; i < arr.length; i++) {
+//     sum += arr[i];
+//     result.push(sum);
+//   }
+//   return result;
+// };
```

Comment:
Great job on this one! This is definitely one of those situations were a for loop would work for us. If you wanted to challenge yourself, you could go back and try a different approach but this is clean and concise code!  👏🏽